.github/workflows/fuzz.yaml

```diff
@@ -0,0 +1,154 @@
+name: Run fuzz tester
+
+on: # Allow manual runs via special PR comment '/run-fuzzer'
+  issue_comment:
+    types: [created, edited]
+  # Allow manual runs via GitHub web interface, CLI etc.
+  workflow_dispatch:
+
+# Allow responding to user by writing a comment on the PR
+permissions:
+  pull-requests: write
```

Comment:
@TD5 Do you have any idea whether it is safe or not? I don't think I want to merge this PR unless we know that it is safe.

Reply:
I am no a GitHub security guru, so I can't give a confidence answer here that addresses security comprehensively. I've updated the PR to specifically limit the permissions of the called workflow in response to your question, because, after consulting the docs, it did seems like the permissions may have been at risk of being inherited by default:

> If `jobs.<job_id>.permissions` is not specified in the calling job, the called workflow will have the default permissions for the GITHUB_TOKEN. For more information, see "[Automatic token authentication](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token)."

_src: https://docs.github.com/en/actions/using-workflows/reusing-workflows#supported-keywords-for-jobs-that-call-a-reusable-workflow_

Reply:
> I am no a GitHub security guru, so I can't give a confidence answer here that addresses security comprehensively.

Neither am I so I would like to err on the side of caution. I don't want for someone to be able to change `base-build.yaml` in their PR and from there do `gh issue edit 1 2 3 4 5 6 7 8 9 10 --body "Cobol is the best"`. I've tried to figure out what permissions GITHUB_TOKEN has per default during the issue_comment event, but I can't find any documentation which makes it clear to me...